File: Excel Sheet Column Title/ESCT.py
class Solution:
    def convertToTitle(self, n: int) -> str:
        arr=["A","B","C","D","E","F","G","H","I","J","K","L","M",
            "N","O","P","Q","R","S","T","U","V","W","X","Y","Z"]
        res = ""
        while(n>26):
            res += arr[n%26-1]
            n = int(n/26) if n%26!=0 else int(n/26)-1
        res +=arr[n-1]
        return res[::-1]

class Solution:
    def convertToTitle(self, n: int) -> str:
        arr=[chr(x) for x in range(ord('A'), ord('Z')+1)]
        res = ""
        while(n>26):
            nn = n%26
            nm =int(n/26)
            res += arr[nn-1]
            n = nm if nn!=0 else nm-1
        res +=arr[n-1]
        return res[::-1]

File: Excel Sheet Column Title/ESCT.java
class ESCT {
    public String convertToTitle(int n) {
        String[] arr=new String[]{"A","B","C","D","E","F","G","H","I","J","K","L","M",
            "N","O","P","Q","R","S","T","U","V","W","X","Y","Z"};
        StringBuilder res = new StringBuilder();
        while(n>26){
            int nn = n%26;
            int nm = n/26;
            res.append(nn==0?arr[arr.length-1]:arr[nn-1]);
            n = nn==0?nm-1:nm;
        }
        res.append(arr[n-1]);
        return res.reverse().toString();
    }

    public static void main(String[] args) {
        ESCT a = new ESCT();
        String res = a.convertToTitle(1);
        System.out.println(res);
    }
}